test/e2e/specs/interactivity/fixtures/index.ts

```diff
@@ -14,12 +14,12 @@ type Fixtures = {
 };
 
 export const test = base.extend< Fixtures >( {
+	// @ts-expect-error: The required type is 'test', but can be 'worker' too. See
+	// https://playwright.dev/docs/test-fixtures#worker-scoped-fixtures
```

Comment:
This `@ts-expect-error` captures a rather large scope. There could be other problems in there and they'd be hidden. The assertion I suggest below is an alternative with a smaller scope. The tradeoff is that it will not be discovered if and when they type if fixed.

```suggestion
```